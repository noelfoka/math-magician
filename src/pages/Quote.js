import React from 'react';

const Quote = () => (
  <div className="Quote">
    <p>
      Mathematics as an expression of the human mind reflects the active
      will, the contemplative reason, and the desire for aesthetic
      perfection. Its basic elements are logic and intuition,
      analysis and construction, generality and individuality. - Richard Courant
    </p>
  </div>
);

export default Quote;
