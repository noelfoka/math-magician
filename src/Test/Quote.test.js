import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

it('renders correctly', () => {
  const quote = renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
