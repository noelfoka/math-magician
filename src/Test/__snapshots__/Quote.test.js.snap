// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Quote"
>
  <p>
    Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. - Richard Courant
  </p>
</div>
`;
